index.html

```diff
@@ -0,0 +1,53 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Apple</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<nav class="top-nav">
+		<ul>
+			<li><i class="fa fa-apple" aria-hidden="true"></i></li>
+			<li><a href=""><span>Store</span></a></li>
+		 <li><a href=""><span>Mac</span></a></li>
+		 <li><a href=""><span>iPod</span></a></li>
+		 <li><a href=""><span>iPhone</span></a></li>
+		 <li><a href=""><span>iPad</span></a></li>
+		 <li><a href=""><span>iTunes</span></a></li>
+		 <li><a href=""><span>Support</span></a></li>
+
+      <li class="searchbox-con"><i class="fa fa-search"></i><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<div class="hero-section">
+				<a href=""><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" alt=""	/></a>
+
+        <a href=""><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" alt="" /></a>
+
+				<a href=""><img	src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" alt=""/></a>
+					</div>
+		</section>
+		<section class="promo-pages">
```

Comment:
Do the same for the images in this container also
Each link is an <a> tag, with a <div> tag inside it.
The <div> tag inside the <a> tag should use an image as background -- [ ] The text on the original page is actually an image